qwik-graphql-tailwind/src/components/repo-header/index.tsx

```diff
@@ -11,17 +11,17 @@ interface RepoHeaderProps {
   stargazerCount: number | string;
   forkCount: number | string;
   watcherCount: number | string;
-  issuesCount: number;
-  prCount: number;
+  issuesCount: number | string;
+  prCount: number | string;
```

Comment:
why did you update this by adding string?
I think it's only appropriate that the counts are numbers. Please reverse.

Reply:
Because we are also, sending it as query params which means they will be strings

Reply:
so if you convert to Number, then why does this have number and string?

Reply:
It is been used in a function that requires it to be a Number.
![image](https://user-images.githubusercontent.com/36667083/203065105-1afe4883-e8e4-40a2-9a49-b7512ca499a4.png)